relentless/optimize/method.py

```diff
@@ -145,15 +153,23 @@ class LineSearch:
 
         \mathbf{d} = \mathbf{x}_{end} - \mathbf{x}_{start}
 
-        \hat{\mathbf{d}} = \frac{\mathbf{d}}{\lVert\mathbf{d}\rVert}
+        \mathbf{\hat{d}} = \frac{\mathbf{d}}{\lVert\mathbf{d}\rVert}
 
     Then, a "target" value :math:`t` is defined as:
 
     .. math::
 
-        t = -\hat{\mathbf{d}} \cdot \nabla{f\left(\mathbf{x}\right)}
+        \nabla f\left(\mathbf{\hat{x}}\right) = \left[{b_1}^2 \frac{\partial f}{\partial x_1},
+                                                      \cdots,
+                                                      {b_n}^2 \frac{\partial f}{\partial x_n}\right]
 
-    Because :math:`\hat{\mathbf{d}}` is a descent direction, the target at the
+        t = -\mathbf{\hat{d}} \cdot \nabla{f\left(\mathbf{\hat{x}}\right)}
+
+    :math:`\mathbf{b}=\left[b_1,\ldots,b_n\right]` is defined as the scaling
+    parameters for the gradient such that :math:`\hat{x}_i=\frac{x_i}{b_i}`. By
+    default, the gradient is left unscaled (:math:`b_i=1`).
```

Comment:
I think we should write this more clearly upfront. The variables are _scaled_, then the optimal step is chosen along the unit vector pointing between the _scaled_ variables.